setup.py

```diff
@@ -32,28 +32,20 @@ def read(*filenames, **kwargs):
 install_req = [
     "jinja2",
     "nengo>=3.1.0",
+    "nengo-extras>=0.5",
```

Comment:
Thinking more about this; do we need to do this? It's only required to use the NengoDL builders, and if someone tries to use those neurons with NengoDL without NengoExtras, they'll get an error.

My biggest concern with this is it's one more dependency that we have to keep up to date both with this and with core Nengo. That said, we *should* be keeping it up to date, so as long as we stay on top of it, I'm OK.

Reply:
We don't need to do it, but not doing makes it super fragile and easy to introduce cyclic dependencies. If you want to remove it and go through all that again you can try, but I would really like to get this release done, and it might require changes to NengoExtras that will need to get released again. NengoExtras itself is not hard to install once Nengo is installed so I'd really prefer to leave it, as it was a big time sink to get it working without cyclic dependencies.

Reply:
I don't quite follow why it would be so fragile. I thought it's just a matter of moving it from `install_req` to `tests_req` in this file (since we do use it for tests). The import in the main code (in `neurons.py`) is already guarded against NengoExtras not being installed.

If it would be hard to change it, though, I'm fine to keep it here. I agree, we want to get the release out.

Reply:
Was easy to change, so I've moved it to "optional" and "tests" :rocket: